azure-pipelines.yml

```diff
@@ -413,7 +413,7 @@ jobs:
         echo "== Test docker image =="
         
         docker load < $TAR_FILE
-        docker run $IMAGE_NAME /bin/bash -c "time az self-test && time az --version && sleep 5"
+        docker run $IMAGE_NAME /bin/bash -c "time az self-test && time az --version && time az bicep version && sleep 5"
```

Comment:
This fails as bicep is not installed.

Reply:
Do you think it's OK to install Bicep as part of this script? I'd really like to verify the end-to-end works here, just to avoid any new compatibility issues in the future.

E.g.
```
docker run $IMAGE_NAME /bin/bash -c "time az self-test && time az --version && time az bicep install && time az bicep version && sleep 5"
```

Reply:
This task is a general test and not meant for specialized testing. All tests should put into the `tests` folder.

In this case, .NET ensures programs can run without `icu` using `DOTNET_SYSTEM_GLOBALIZATION_INVARIANT=1`. I think adding a test to ensures the `az bicep version` still works with this env is sufficient.

Ref: https://learn.microsoft.com/en-us/dotnet/core/runtime-config/globalization#invariant-mode

Reply:
Makes sense. In this case I wasn't so worried about globalization - was more just thinking about what else could go wrong in the future if other capabilities are stripped out of the container image.